llama_stack/providers/registry/safety.py

```diff
@@ -27,27 +27,6 @@ def available_providers() -> List[ProviderSpec]:
             module="llama_stack.providers.inline.safety.prompt_guard",
             config_class="llama_stack.providers.inline.safety.prompt_guard.PromptGuardConfig",
         ),
-        InlineProviderSpec(
```

Comment:
a really old deprecation, removing this. the test flagged it saying the module was not there!